src/veins/base/phyLayer/BasePhyLayer.cc

```diff
@@ -15,6 +15,7 @@
 
 using namespace Veins;
 
+using std::unique_ptr;
```

Comment:
Suggestion: Drop this and simply qualify `std::unique_ptr` where used. Even it gets inconsistent pretty fast otherwise.

Reply:
Since the `unique_ptr`'s are now used widely in this file I prefer this way to remove some clutter. It is inconsistent with the header, since `using`s in the header would pollute the public namespace, but I still think it's worth it to save redundant information.